docs/how-to-guides/themes/create-block-theme.md

```diff
@@ -39,339 +40,822 @@ theme
+- Copy template part one and two from `theme/block-template-parts/` to your theme's `block-template-part` folder.
+- Rename the template parts to `header.html` and `footer.html`, respectively.
+- Open `index.html` and update the template part slugs in the block markup.
+
+Saved templates have precedence over theme files. To use the updated theme files, go to **Appearance > Templates** and
+**Appearance > Template parts** and delete the saved templates.
+
+### Additional templates
+
+#### Blog
+
+Now the theme has a basic site header and footer, but it does not display any content.
+To create a list of posts, you will use the query and query loop blocks.
+
+Wether you are using the site editor or editing theme files directly, open the index template.
+
+First, add a group block that will work as a container for the posts.
+Next, enable the width options for the blocks inside this group using `"layout":{"inherit":true}`.
+
+```html
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"></div>
+<!-- /wp:group -->
+<!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
 ```
+
+Change the `<div>` in the group block to a `<main>` element using the `tagName` attribute:
+
+```html
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+<!-- wp:group {"layout":{"inherit":true},"tagName":"main"} -->
+<main class="wp-block-group"></main>
+<!-- /wp:group -->
+<!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
+```
+
+If you are using one of the editors, change the element from `<div>` to `<main>` under **Advanced** in the block setting sidebar.
+
+Add a query block inside the group.
+When you place a query block in the editor, the query loop is used as an inner block and you have the option to start with an empty loop or include selected post blocks like a post title and featured image.
+
+Example markup:
+
+```html
+<!-- wp:query -->
+<div class="wp-block-query"><!-- wp:query-loop -->
+<!-- wp:post-title /-->
+<!-- wp:post-date /-->
+<!-- wp:post-excerpt /-->
+<!-- /wp:query-loop --></div>
+<!-- /wp:query -->
+```
+
+The query pagination block can only be used inside the query. Place it inside the query, but outside the loop:
+
+```html
+<!-- wp:query -->
+<div class="wp-block-query"><!-- wp:query-loop -->
+<!-- wp:post-title /-->
+<!-- wp:post-date /-->
+<!-- wp:post-excerpt /-->
+<!-- /wp:query-loop -->
+
+<!-- wp:query-pagination -->
+<div class="wp-block-query-pagination">
+<!-- wp:query-pagination-previous /-->
+<!-- wp:query-pagination-numbers /-->
+<!-- wp:query-pagination-next /--></div>
+<!-- /wp:query-pagination -->
+
+</div>
+<!-- /wp:query -->
+```
+
+#### Posts and pages
+
+Next, create a new template for displaying single posts.
+If you are editing theme files directly, create a file called `single.html` inside the block-templates folder.
```

Comment:
Should this be `singular.html` since it is for posts and pages?

Reply:
Blow this, you are asked to create a second file/template for pages.
It is just to practice creating the templates. A page template will have different blocks, for example you might not want to have an author and next and previous navigation.